src/components/tabs/tabs.scss

```diff
@@ -167,6 +167,11 @@ md-pagination-wrapper {
     width: auto;
     margin: 0 auto;
   }
+  &.md-align-end {
+    position: relative;
+    width: auto;
+    margin: 0 0 0 auto; 
```

Comment:
P.S.: Tabs do not work well with rtl. I'm guessing that won't be a good option to go with.